Cargo.toml

```diff
@@ -13,16 +13,17 @@ version = "0.8.0"
 brotli2 = "~0.2.1"
 memmap = "~0.4.0"
 rand = "~0.3.14"
-rust_sodium = "~0.2.0"
-serde = "~0.9.12"
-serde_derive = "~0.9.12"
+rust_sodium = { git = "https://github.com/maqi/rust_sodium", branch = "update" }
```

Comment:
updated to crates, pls go through the rest of the file - nothing should poitn to github anymore.